about.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
you can put this file and your other html pages into a directory called pages to organize your files a little more